p4-16/spec/P4-16-draft-spec.mdk

```diff
@@ -4876,22 +4877,18 @@ match_kind {
 ~ End P4Example
 
 These identifiers correspond to the P4~14~ match kinds with the same
-names. The semantics of these annotations is actually irrelevant for
-describing the behavior of the P4 abstract machine; how they are used
-influences only the control-plane API and the actual implementation of
-the look-up table. From the point of view of the P4 program, a look-up
-table is an abstract finite map that is given a key and produces as a
-result either an action or a "miss" indication, as described in
-Section [#sec-mau-semantics].
```

Comment:
Yes, the data-plane does not care about the match_kind; it does not appear in the definition of the interface I described above.